infra/helper.py

```diff
@@ -223,9 +226,9 @@ def _get_corpus_dir(project_name=''):
   return os.path.join(BUILD_DIR, 'corpus', project_name)
 
 
-def _get_output_dir(project_name=''):
+def _get_output_dir(project_name='', sanitizer=''):
```

Comment:
I think we need engine as well.

/out/libpng/libfuzzer-msan 
/out/libpng/afl-asan